samples/azure/webjob-host/sample.md

```diff
@@ -12,33 +12,33 @@ related:
 
 ---
 
-This sample is compatible with WebJobs SDK 3.0
+This sample is compatible with Azure WebJobs SDK 3.0.
 
 ## Running in development mode
 
- 1. Start [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/storage-use-emulator)
- 1. Run the solution
+ 1. Start the [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/storage-use-emulator).
+ 1. Run the solution.
 
 ## Code walk-through
 
 This sample contains one project:
 
-- Receiver - Self-hosted endpoint running in a continuous web job
+- Receiver: A self-hosted endpoint running in a continuous WebJob.
 
-### Receiver project
+### Receiver
 
-The receiver project uses the self-hosting capability to start an end endpoint inside a continuously running web job.
+The receiver uses the self-hosting capability to start an end endpoint inside a continuously running WebJob.
 
-The snippet below illustrates how the `StartAsync` method of `IJobHost` can be used to configure and start the endpoint
+The `StartAsync` method of `IJobHost` is used to configure and start the endpoint:
 
 snippet: WebJobHost_Start
 
-If dependencies need to be shared between the service collection and NServiceBus infrastructure like handlers the [MSDependencyInjection nuget package](/nservicebus/dependency-injection/msdependencyinjection.md) needs to be configured.
+NOTE: If dependencies need to be shared between the service collection and NServiceBus infrastructure, such as message handlers, the [MSDependencyInjection NuGet package](/nservicebus/dependency-injection/msdependencyinjection.md) must be installed.
```

Comment:
@danielmarbach I wasn't really sure what you meant by "configured" here. Did you mean "installed"?

Reply:
No I meant properly configured. Just installing it is not enough. Because it needs to be setup to share the container but I guess installed is good enough